src/presentation/routers/article-router.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response } from 'express';
+import NoSQLWrapper from '../../data/interfaces/data-sources/no-sql-wrapper';
+
+const ArticleRouter = (db: NoSQLWrapper) => {
+    const router = express.Router();
+    router.get('/articles/recent', async (req: Request, res: Response) => {
+    try {
+    const page = parseInt(req.query.page as string) || 1;
+    const pageSize = parseInt(req.query.pageSize as string) || 10;
+    const recentArticles = await getRecentArticlesMock(page, pageSize);
+    res.json(recentArticles);
+    } catch (error) {
+    console.error(error);
+    res.status(500).json({ error: 'Internal Server Error' });
```

Comment:
Use the code state from the library https://www.npmjs.com/package/http-status-codes